start.sh

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash
 
-PRINTMODE=""
-while getopts ":hp:" OPTION
-do
-    case $OPTION in
-        h)
-            /launch_df -h
-            exit 1
-            ;;
-        p)
-            PRINTMODE=$OPTARG
-            ;;
-    esac
-done
 
 /usr/bin/supervisord
+set -x
 
-# Enable auto backups using gitwatch of the save directory is a git repo
```

Comment:
I also think it is more like an "additional feature" rather than part of the core functionality, so before I started looking into ansible I was thinking of a way to make this an optional plugin anyway. So I have no objection for removing it.